src/lib.rs

```diff
@@ -103,6 +103,8 @@ mod transport;
 
 pub use self::error::Error;
 pub use self::filter::Filter;
+#[cfg(feature = "compression")]
```

Comment:
Didn't notice, can you mark this `doc(hidden)` and move it below the comments why?

Reply:
Updated and moved